core/src/wallet/bitcoin/BitcoinLikeAccount.cpp

```diff
@@ -531,14 +531,32 @@ namespace ledger {
             });
         }
 
+        // This method is used to get a block height only used to know which
+        // update/fork (activated at certain block height) is used.
+        // This explains why we take LLONG_MAX is we have no block in DB for
+        // certain currency
+        // WARNING: please don't use this method if you need an accurate value
+        // of last block
+        static uint64_t getLastBlockFromDB(soci::session &sql, const std::string &currencyName) {
+            //Get last block from DB
+            auto lastBlock= BlockDatabaseHelper::getLastBlock(sql, currencyName);
```

Comment:
Whitespace between `lastBlock` and `=` :)

Reply:
My bad !